webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -136,8 +140,8 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
       this.contactsActions.removeContactFromList({ _id: change.id });
       this.hasContacts = !!this.contactsList.length;
     }
-    if (this.usersHomePlace && change.id === this.usersHomePlace._id) {
-      this.usersHomePlace = await this.getUserHomePlaceSummary(change.id);
+    if (this.usersHomePlace?.some(homePlace => homePlace._id === change.id)) {
```

Comment:
I think `find` is a better choice here than `some`, because `find` will stop at the first found match, which `some` will continue iterating even after finding one match.